src/main/webapp/WEB-INF/view/ActivityFeed.jsp

```diff
@@ -13,45 +13,58 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+<%@ page import="java.util.List"%>
+<%@ page import="codeu.model.data.Conversation"%>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
 <!DOCTYPE html>
 <html>
 <head>
-  <title>ActivityFeed</title>
-  <link rel="stylesheet" href="/css/main.css">
+<title>ActivityFeed</title>
+<link rel="stylesheet" href="/css/main.css">
 </head>
 <body>
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
-    <a href="/conversations">Conversations</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else{ %>
-      <a href="/activityfeed">Login</a>
-    <% } %>
-    <a href="/about.jsp">About</a>
+  <a id="navTitle" href="/">CodeU Chat App</a> <a href="/conversations">Conversations</a>
+  <% if (request.getSession().getAttribute("user") != null) { %>
+  <a>Hello <%=request.getSession().getAttribute("user")%>!</a>
```

Comment:
Want to maintain indents like the code used to here. So that you can tell what logic is within the if/else statements.

Reply:
I think I fixed

Reply:
`
<% if (request.getSession().getAttribute("user") != null) { %>
  <a>Hello <%=request.getSession().getAttribute("user")%>!</a>
<% } else { %>
`

Reply:
The <a> should be indented more than the if statment, same for the code between the {} of the else